### PR TITLE
Turbo tweaks

### PR DESCRIPTION
**Pull request by petebacondarwin**

### CODE REVIEW

**Review by rozenmd — APPROVED** [review]
Looks good to me.

File: fixtures/remix-pages-app/package.json [hunk 1]
```diff
@@ -15,23 +15,23 @@
 		"test:watch": "npx vitest"
 	},
 	"dependencies": {
-		"@remix-run/cloudflare-pages": "^1.8.2",
+		"@remix-run/cloudflare-pages": "1.16.1",
```
Comment (GregBrimble) [inline fixtures/remix-pages-app/package.json:18]: 1.17.0?

File: .changeset/flat-badgers-kiss.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+"no-bundle-import": patch
+"node-app-pages": patch
+"pages-workerjs-app": patch
+"pages-workerjs-directory": patch
+"remix-pages-app": patch
```
Comment (GregBrimble) [inline .changeset/flat-badgers-kiss.md:6]: Do we usually changeset the fixtures? I've never seen us do that before.

**Review by GregBrimble — APPROVED** [review]
Couple of questions

File: .changeset/flat-badgers-kiss.md (hunk 2 above)
Comment (petebacondarwin) [reply .changeset/flat-badgers-kiss.md:6]: Yeah ... good question. Happy to go either way.

File: fixtures/remix-pages-app/package.json (hunk 1 above)
Comment (petebacondarwin) [reply fixtures/remix-pages-app/package.json:18]: I was just trying to get it working with as little churn as possible; and the current highest version that was in the package.json was 1.16.2.
Feel free to bump to 1.17.0 in a follow up.

Comment (petebacondarwin) [reply fixtures/remix-pages-app/package.json:18]: Huh? I thought made them all 1.16.2, right? Which ones are inconsistent?

Comment (petebacondarwin) [reply fixtures/remix-pages-app/package.json:18]: OMG! I just saw what you mean.

File: .changeset/flat-badgers-kiss.md (hunk 2 above)
Comment (petebacondarwin) [reply .changeset/flat-badgers-kiss.md:6]: Removing

**Review by GregBrimble — APPROVED** [review]

**Review by JacobMGEvans — APPROVED** [review]
Amazing!

**Pull request merged**